usr/sbin/regionsrv-enabler-azure

```diff
@@ -86,8 +105,9 @@ elif license_type and 'BYOS' not in license_type:
     # or PAYG without AHB
     update_server = utils.get_smt()
     if update_server:
-        if utils.is_registered(update_server.get_FQDN()):
```

Comment:
Not the only change needed, we also have

`if 'BYOS' in license_type and not utils.uses_rmt_as_scc_proxy():`

(line 100)

currently this means that the commands in the if condition get executed every time the timer runs and the condition is met. However now that we have a cache we can optimize this behavior and stop sending so many messages to the systemlog.

That means the new condition is:

`if 'BYOS' in license_type and has_license_changed(license_type) and not utils.uses_rmt_as_scc_proxy():`

Reply:
True, I though it would not be a big issue to clean every BYOS attempt, updating it now